[Bug] [Build] build fail, due to slf4j-reload4j:1.7.25 missing in maven repo

### Search before asking

- [x] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

[ERROR] Failed to execute goal on project seatunnel-flink-connector-v2-example: Could not resolve dependencies for project org.apache.seatunnel:seatunnel-flink-connector-v2-example:jar:2.3.9-SNAPSHOT: org.slf4j:slf4j-reload4j:jar:1.7.25 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[E

### SeaTunnel Version

2.3.9

### SeaTunnel Config

```conf
none
```

### Running Command

```shell
none
```

### Error Exception

```log
[ERROR] Failed to execute goal on project seatunnel-flink-connector-v2-example: Could not resolve dependencies for project org.apache.seatunnel:seatunnel-flink-connector-v2-example:jar:2.3.9-SNAPSHOT: org.slf4j:slf4j-reload4j:jar:1.7.25 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[E
```

### Zeta or Flink or Spark Version

_No response_

### Java or Scala Version

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/pom.xml
+++ b/pom.xml
@@ -66,7 +66,7 @@
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
 
-        <slf4j.version>1.7.25</slf4j.version>
+        <slf4j.version>1.7.36</slf4j.version>
         <log4j2.version>2.17.1</log4j2.version>
         <log4j2-disruptor.version>3.4.4</log4j2-disruptor.version>
         <log4j.version>1.2.17</log4j.version>

